targets/esp8266/Makefile

```diff
@@ -73,32 +75,42 @@ LINKFLAGS_eagle.app.v6 =				\
 	-u call_user_start				\
 	-Wl,-static					\
 	-Wl,--start-group				\
-	-lminic						\
+	-lcirom						\
+	-lcrypto					\
+	-lespconn					\
+	-lespnow					\
+	-lfreertos					\
 	-lgcc						\
 	-lhal						\
+	-ljson						\
+	-llwip						\
+	-ldriver					\
+	-lmain						\
+	-lmesh						\
+	-lmirom						\
+	-lnet80211					\
+	-lnopoll					\
 	-lphy						\
 	-lpp						\
-	-lnet80211					\
+	-lpwm						\
+	-lsmartconfig					\
+	-lspiffs					\
+	-lssl						\
 	-lwpa						\
-	-lmain						\
-	-lfreertos					\
-	-llwip						\
+	-lwps						\
 	-L./libs					\
-	-ljerrycore					\
-	-ljerryentry					\
-	-ljerrylibm					\
+	-ljerry-core					\
+	-ljerry-libm					\
 	$(DEP_LIBS_eagle.app.v6)			\
-	./libs/lib_a-setjmp.o				\
 	-Wl,--end-group
 
 
 DEPENDS_eagle.app.v6 =					\
 	$(LD_FILE)					\
 	$(LDDIR)/eagle.rom.addr.v6.ld			\
```

Comment:
are these two lines OK? the linker script is copied from the sdk and modified here (LDDIR is `$(SDK_PATH)/ld` if I'm not mistaken), and LD_FILE is defined only _after_ this line.

Reply:
The second line is OK, the first line needs a bit modification as you pointed to this problem.